lib/babe/babe.go

```diff
@@ -386,6 +386,7 @@ func (b *Service) handleEpoch(epoch uint64) (next uint64, err error) {
 		// stop current epoch handler
 		cancel()
 	case err := <-errCh:
+		// TODO: errEpochPast is sent on this channel, but it doesnot get logged here
```

Comment:
Not sure what you mean here, the error is logged below no?

Reply:
I have seen below part getting executed 
https://github.com/ChainSafe/gossamer/blob/67a9bbbc4d840858000aabb486438db2d7d435aa/lib/babe/epoch_handler.go#L73-L76

Because `errEpochPast` is being sent on the channel here, I should see `error from epochHandler` in logs because of following code
https://github.com/ChainSafe/gossamer/blob/67a9bbbc4d840858000aabb486438db2d7d435aa/lib/babe/babe.go#L388-L390

I did not see `error from epochHandler` in the logs when I was running gossamer

Reply:
@kishansagathiya can you open an issue for this?